From 189b3c00e233a967c948fed316908df60446d32e Mon Sep 17 00:00:00 2001
From: John Galt <johngaltfirstrun@gmail.com>
Date: Tue, 9 Dec 2025 07:53:37 -0500
Subject: [PATCH 5/5] fix all apps blur

getBottomSheetDepth is returning 0 due to old changes or mismerge. This
is temporary
---
 .../uioverrides/states/AllAppsState.java      | 20 ++++++++-----------
 1 file changed, 8 insertions(+), 12 deletions(-)

diff --git a/quickstep/src/com/android/launcher3/uioverrides/states/AllAppsState.java b/quickstep/src/com/android/launcher3/uioverrides/states/AllAppsState.java
index 553408afadc..29751ecc34a 100644
--- a/quickstep/src/com/android/launcher3/uioverrides/states/AllAppsState.java
+++ b/quickstep/src/com/android/launcher3/uioverrides/states/AllAppsState.java
@@ -134,19 +134,15 @@ public class AllAppsState extends LauncherState {
 
     @Override
     protected <DEVICE_PROFILE_CONTEXT extends Context & ActivityContext>
-            float getDepthUnchecked(DEVICE_PROFILE_CONTEXT context) {
-        if (context.getDeviceProfile().shouldShowAllAppsOnSheet()) {
-            return context.getDeviceProfile().getBottomSheetProfile().getBottomSheetDepth();
+        float getDepthUnchecked(DEVICE_PROFILE_CONTEXT context) {
+        // The scrim fades in at approximately 50% of the swipe gesture.
+        if (enableScalingRevealHomeAnimation()) {
+            // This means that the depth should be twice of what we want, in order to fully zoom
+            // out during the visible portion of the animation.
+            return BaseDepthController.DEPTH_60_PERCENT;
         } else {
-            // The scrim fades in at approximately 50% of the swipe gesture.
-            if (enableScalingRevealHomeAnimation()) {
-                // This means that the depth should be twice of what we want, in order to fully zoom
-                // out during the visible portion of the animation.
-                return BaseDepthController.DEPTH_60_PERCENT;
-            } else {
-                // This means that the depth should be greater than 1, in order to fully zoom out.
-                return 2f;
-            }
+            // This means that the depth should be greater than 1, in order to fully zoom out.
+            return 2f;
         }
     }
 
-- 
2.52.0

